docs/tutorials/Filter_tags.md

```diff
@@ -0,0 +1,65 @@
+|filter tag|transaction field|
```

Comment:
Could you update the `go:generate` command in `searcher.go` so that the result is written to this path?

Reply:
updated.